tests/test_benchmarking/test_data/test_simulated_data/test_mock_student_provider.py

```diff
@@ -21,6 +21,76 @@ def test_get__returns_student_objects(self):
         for student in students:
             self.assertIsInstance(student, Student)
 
+    def test_get__each_student_has_correct_num_friends_and_enemies(self):
+        students = MockStudentProvider(
+            MockStudentProviderSettings(
+                number_of_students=10,
+                number_of_friends=4,
+                number_of_enemies=1,
+            )
+        ).get()
+
+        for student in students:
+            num_friends = len(
+                [
+                    1
+                    for stu_id, relation in student.relationships.items()
+                    if relation == Relationship.FRIEND
+                ]
+            )
+            num_enemies = len(
+                [
+                    1
```

Comment:
nit; make this 2, I think the way it worked before could have still passed this test if you use 1